app.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    $('input#addthought').click(function() {
```

Comment:
Since you added ID's to the elements, you could have just used them as your selector.